packages/statemanager/src/util.ts

```diff
@@ -1,7 +1,32 @@
+import { VerkleAccessedStateType } from '@ethereumjs/common'
 import { Account, bytesToHex } from '@ethereumjs/util'
 
 import type { AccountFields, StateManagerInterface } from '@ethereumjs/common'
-import type { Address } from '@ethereumjs/util'
+import type { Address, PrefixedHexString } from '@ethereumjs/util'
+
+/**
+ * Helper method for decoding verkle accessWitness values
+ * @param type the type of the accessed state
+ * @param value the value of the accessed state
+ * @returns string - the decoded value of the accessed state
+ */
+export function decodeVerkleAccessWitnessValue(
+  type: VerkleAccessedStateType,
+  value: PrefixedHexString | null,
+): string {
+  if (value === null) {
+    return ''
+  }
+
+  switch (type) {
```

Comment:
Is there an expectation that we might change how each one is handled in the future?  This seems like a lot of lines of code for essentially `value === null ? '' : value`

Reply:
This is really just a relic from the time where we had separate treatment for each. I'll remove it, it brings no value at this time.

Reply:
I've simplified this part and the usage upstream. ty for the suggestion